### PR TITLE
renaming from vata2 to mata

### PR DESCRIPTION
**Pull request by martinhruska**
This pull request:

- changes namespaces from Vata2 to Mata as well as the names of files, etc.,
- changes any references to Vata2 and renames it to Mata.

I don't think there is much to review but i leave it here for a day so you can check changes if you want.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
-git clone https://github.com/ondrik/libvata2
-cd libvata2
+git clone https://github.com/ondrik/libmata
+cd libmata
 make release
 ```
 
 # Documentation
-see [this page](https://ondrik.github.io/libvata2/api)
+see [this page](https://ondrik.github.io/libmata/api)
```
Comment (Adda0) [inline README.md:16]: This link will not work.

File: README.md [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
 
-An attempt to revamp VATA
+An attempt to revamp MATA
 
 # Building
 
 ```
-git clone https://github.com/ondrik/libvata2
-cd libvata2
+git clone https://github.com/ondrik/libmata
```
Comment (Adda0) [inline README.md:10]: The whole link should be updated to point to https://github.com/VeriFIT/mata.

File: README.md [hunk 3]
```diff
@@ -1,16 +1,16 @@
...
+cd libmata
 make release
```
Comment (Adda0) [inline README.md:12]: The installation instructions should be updated as well.

**Review by Adda0 — COMMENTED** [review]
Other than the mentioned, LGTM.

File: README.md (hunk 1 above)
Comment (martinhruska) [reply README.md]: thank you for feedback. Resolved.

**Pull request merged**